Assets/Scripts/SceneManagement/DistanceTrigger.cs

```diff
@@ -0,0 +1,23 @@
+﻿using UnityEngine;
+
+namespace Anivision.SceneManagement
+{
+    // A SceneSwitch script that triggers when this script's game object gets withen the distance threshold to the given headset transform
+    public class DistanceTrigger : SceneSwitch
```

Comment:
Just curious - Why trigger through distance instead of through onTriggerEnter?

Reply:
Its a bit of a remnant from the original script, but I also thought it might be better for cases where we want the scene to switch when the user gets to a certain point in scene. (and this script would be applied to some stationary, possibly invisible marker)

Reply:
ok makes sense